source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -499,6 +500,26 @@ int StreamHandleWrapper::luaVerifySignature(lua_State* state) {
   return 2;
 }
 
+int StreamHandleWrapper::luaTimestamp(lua_State* state) {
+  auto now = time_source_.systemTime().time_since_epoch();
+
+  auto time_unit = luaL_optstring(state, 2, "");
```

Comment:
i think you can do `absl::string_view unit_parameter = ...` and not have to use the strcmp helpers, relying on `empty()` and `operator==`

that said id be curious to see if @dio's suggestion makes things cleaner, wdyt?

Reply:
Haven't quite figured out the enum thing, pushed `absl::string_view unit_paramter...` change in the meantime.